libs/features/jarvis-chat.js

```diff
@@ -217,8 +218,8 @@ const startInitialization = async (config, event) => {
   }
 
   window.AdobeMessagingExperienceClient.initialize({
-    appid: config.jarvis.id,
-    appver: config.jarvis.version,
+    appid: getMetadata('jarvis-surface-id') || config.jarvis.id,
```

Comment:
Are there different surface-ids across a project?

Reply:
Not right now, but that's the requirement here - be able to initialise Jarvis using different appid, which in turn will run different workflows / experiences.